From 4f52d718ba1631389c90f7900603d7465148f47b Mon Sep 17 00:00:00 2001
From: Matthew Shyu <matthew.shyu@amlogic.com>
Date: Thu, 28 Dec 2017 17:39:33 +0800
Subject: [PATCH] Modify gxl_p230_2g.dts and gxl_p212_2g.dts for DRM backend

Make most part of dts the same as that of p212
---
 arch/arm64/boot/dts/amlogic/gxl_p212_2g.dts | 2 +-
 arch/arm64/boot/dts/amlogic/gxl_p230_2g.dts | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

Index: common/arch/arm64/boot/dts/amlogic/gxl_p212_2g.dts
===================================================================
--- common.orig/arch/arm64/boot/dts/amlogic/gxl_p212_2g.dts	2019-02-14 18:42:00.110175128 +0800
+++ common/arch/arm64/boot/dts/amlogic/gxl_p212_2g.dts	2019-02-14 18:43:54.034177265 +0800
@@ -98,7 +98,7 @@
 		ion_reserved:linux,ion-dev {
 			compatible = "shared-dma-pool";
 			reusable;
-			size = <0x0 0x7C00000>;
++			size = <0x0 0x8000000>;
 			alignment = <0x0 0x400000>;
 		};
 
@@ -756,7 +756,7 @@
 		compatible = "amlogic, meson-gxl";
 		memory-region = <&logo_reserved>;
 		dev_name = "meson-fb";
-		status = "okay";
+		status = "disabled";
 		interrupts = <0 3 1
 			0 89 1>;
 		interrupt-names = "viu-vsync", "rdma";
Index: common/arch/arm64/boot/dts/amlogic/gxl_p230_2g.dts
===================================================================
--- common.orig/arch/arm64/boot/dts/amlogic/gxl_p230_2g.dts	2019-02-14 18:42:00.110175128 +0800
+++ common/arch/arm64/boot/dts/amlogic/gxl_p230_2g.dts	2019-02-14 18:43:36.782176942 +0800
@@ -97,7 +97,7 @@
 		ion_reserved:linux,ion-dev {
 			compatible = "shared-dma-pool";
 			reusable;
-			size = <0x0 0x7C00000>;
++			size = <0x0 0x8000000>;
 			alignment = <0x0 0x400000>;
 		};
 
@@ -745,7 +745,7 @@
 		compatible = "amlogic, meson-gxl";
 		memory-region = <&logo_reserved>;
 		dev_name = "meson-fb";
-		status = "okay";
+		status = "disabled";
 		interrupts = <0 3 1
 			0 89 1>;
 		interrupt-names = "viu-vsync", "rdma";
